pkg/server/api/bundle/v1/service.go

```diff
@@ -0,0 +1,158 @@
+package bundle
+
+import (
+	"context"
+	"errors"
+
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	"github.com/spiffe/spire/pkg/server/api/rpccontext"
+	"github.com/spiffe/spire/pkg/server/plugin/datastore"
+	"github.com/spiffe/spire/proto/spire-next/api/server/bundle/v1"
+	"github.com/spiffe/spire/proto/spire-next/types"
+	"github.com/spiffe/spire/proto/spire/common"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// RegisterService registers the bundle service on the gRPC server.
+func RegisterService(s *grpc.Server, service *Service) {
+	bundle.RegisterBundleServer(s, service)
+}
+
+// Config is the service configuration
+type Config struct {
+	Datastore   datastore.DataStore
+	TrustDomain spiffeid.TrustDomain
+}
+
+// New creates a new bundle service
+func New(config Config) *Service {
+	return &Service{
+		ds: config.Datastore,
+		td: config.TrustDomain,
+	}
+}
+
+// Service implements the v1 bundle service
+type Service struct {
+	ds datastore.DataStore
+	td spiffeid.TrustDomain
+}
+
+func (s *Service) GetBundle(ctx context.Context, req *bundle.GetBundleRequest) (*types.Bundle, error) {
+	return nil, status.Errorf(codes.Unimplemented, "method GetBundle not implemented")
+}
+
+func (s *Service) AppendBundle(ctx context.Context, req *bundle.AppendBundleRequest) (*types.Bundle, error) {
+	return nil, status.Errorf(codes.Unimplemented, "method AppendBundle not implemented")
+}
+
+func (s *Service) ListFederatedBundles(ctx context.Context, req *bundle.ListFederatedBundlesRequest) (*bundle.ListFederatedBundlesResponse, error) {
+	return nil, status.Errorf(codes.Unimplemented, "method ListFederatedBundles not implemented")
+}
+
+func (s *Service) GetFederatedBundle(ctx context.Context, req *bundle.GetFederatedBundleRequest) (*types.Bundle, error) {
+	log := rpccontext.Logger(ctx).WithField("RPC", "GetFedertedBundle")
```

Comment:
This is handled by the logging middleware aleady (it adds the service and method fields)

Reply:
Great, I'll remove it!